comp/process/apiserver/apiserver_test.go

```diff
@@ -43,13 +50,54 @@ func TestLifecycle(t *testing.T) {
 		fetchonlyimpl.MockModule(),
 	))
 
-	assert.Eventually(t, func() bool {
-		res, err := http.Get("http://localhost:6162/config")
-		if err != nil {
-			return false
-		}
+	assert.EventuallyWithT(t, func(c *assert.CollectT) {
+		req, err := http.NewRequest("GET", "http://localhost:43424/agent/status", nil)
+		require.NoError(c, err)
+		util.CreateAndSetAuthToken(pkgconfigsetup.Datadog())
```

Comment:
As explain earlier, if it set before, no need to set it again.

```suggestion
```

Reply:
Removed